grakn-graph/src/test/java/ai/grakn/graph/property/RelationPropertyTest.java

```diff
@@ -44,6 +48,41 @@
     public final ExpectedException exception = ExpectedException.none();
 
     @Property
+    public void whenAddingARolePlayer_ItIsAddedToTheCollectionOfRolePlayers(
+        Relation relation, @FromGraph RoleType roleType, @FromGraph Instance rolePlayer) {
+
+        assumeFalse(rolePlayer.isResource());
```

Comment:
Actually this check can also be removed in all these tests, I suspect. Sorry, wasn't clear!